ocaml/tests/test_clustering.ml

```diff
@@ -293,15 +306,10 @@ let test_assert_pif_prerequisites () =
         failwith exn);
   Db.PIF.set_disallow_unplug ~__context ~self:pifref ~value:true;
   let pif = Xapi_clustering.pif_of_host ~__context network localhost in
-  OUnit.assert_equal (Xapi_clustering.assert_pif_prerequisites pif) ()
-
-let test =
-  let open OUnit in
-  "test_clustering" >:::
-  [ test_get_required_cluster_stacks
-  ; test_find_cluster_host
-  ; test_assert_cluster_host_enabled
-  ; test_assert_cluster_host_is_enabled_for_matching_sms
-  ; test_clustering_lock_only_taken_if_needed
-  ; "test_assert_pif_prerequisites" >:: test_assert_pif_prerequisites
-  ]
+  Alcotest.(check unit)
+    "assert_pif_prerequisites should pass after setting disallow_unplug:true"
+    (Xapi_clustering.assert_pif_prerequisites pif) ()
+
+let test_assert_pif_prerequisites =
+  [ "test_assert_pif_prerequisites", `Quick, test_assert_pif_prerequisites ]
```

Comment:
I think we should have all the tests in this file in a `tests` variable, like in `test_vdi_cbt.ml`, otherwise the main `alcotest_suite.ml` will get huge:
```
let tests =
  test_assert_pif_prerequisites @
  test_clustering_lock_only_taken_if_needed @ ...
```
And we should ensure that the names of the above tests are specific enough, because alcotest does not do the subgroup namespacing done by OUnit, because it does not have nested test groups.